docker_daemon/README.md

```diff
@@ -186,7 +186,7 @@ Need help? Contact [Datadog support][22].
 [18]: https://docs.datadoghq.com/agent/#cli
 [19]: https://github.com/DataDog/integrations-core/blob/master/docker_daemon/metadata.csv
 [20]: https://github.com/DataDog/integrations-core/blob/master/docker_daemon/assets/service_checks.json
-[21]: https://github.com/DataDog/integrations-core/blob/master/docker_daemon/datadog_checks/docker_daemon/data/conf.yaml.example#L124
+[21]: https://github.com/DataDog/integrations-core/blob/master/docker_daemon/datadog_checks/docker_daemon/data/conf.yaml.example#L154
```

Comment:
```suggestion
[21]: https://github.com/DataDog/integrations-core/blob/master/docker_daemon/datadog_checks/docker_daemon/data/conf.yaml.example#L151
```

Reply:
I think we shouldn’t link to a particular line in a file on the `master` branch because the file on `master` can evolve so that the line isn’t pointing anymore to what we originally wanted.
In order to link to a particular line, we should point to a fixed version of the file.

```suggestion
[21]: https://github.com/DataDog/integrations-core/blob/7.39.0/docker_daemon/datadog_checks/docker_daemon/data/conf.yaml.example#L151-L154
```